src/main/java/org/opensearch/jobscheduler/rest/action/RestGetAllLocksAction.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.rest.action;
+
+import org.opensearch.transport.client.node.NodeClient;
+import org.opensearch.jobscheduler.JobSchedulerPlugin;
+import org.opensearch.jobscheduler.transport.action.GetAllLocksAction;
+import org.opensearch.jobscheduler.transport.request.GetAllLocksRequest;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestToXContentListener;
+
+import java.util.List;
+
+import static org.opensearch.rest.RestRequest.Method.GET;
+
+/**
+ * REST handler for getting all locks
+ */
+public class RestGetAllLocksAction extends BaseRestHandler {
```

Comment:
nit: let's remove `All` from all related classes. I think this can be shortened to `RestGetLocksAction`